src/docs/operator-guides/migrating-from-outside-sharetribe/index.md

```diff
@@ -109,12 +215,202 @@ key like this:
+  - `:im.image/listing`
+  - `:im.image/sortOrder`
+
+Supported references
+
+- `#im.image/listing` -> `#im.listing`
+
+Note that for profile images, the reference is added in [user](#user)
+with `#im.userProfile/avatar` -> `#im.image`
+
+Example syntax
+
+```
+[[:im.image/id #uuid "58afd8e1-e336-4ca4-a1e7-ff1d91856a6c"]
+  #:im.image{:url "https://asset-url.someservice.com/path/to/img1.jpg"
+            :sortOrder 1
+            :listing #im/ref :listing/rock-sauna}]
+```
+
+#### Stripe account
+
+- Required attributes
+  - `:im.stripeAccount/stripeAccountId`
+  - `:im.stripeAccount/user` TODO: how about chargesEnabled and
+    payoutsEnabled?
+
+Example syntax
+
+```
+[[:im.stripeAccount/id]
+  #:im.stripeAccount{:stripeAccountId "a_stripe_id"
+                    :chargesEnabled true
+                    :payoutsEnabled false
+                    :user #im/ref :user/john}]
+```
+
+#### Review
+
+Supported fields to migrate:
+[API reference review resource](https://www.sharetribe.com/api-reference/marketplace.html#review-resource-format)
+
+There are two types of reviews: `ofCustomer` and `ofProvider`. They
+differ in the reference to listing, as `ofProvider` reviews have a
+reference to a listing and `ofCustomer` don't.
+
+- Required attributes
+  - `:im.review/type`
+  - `:im.review/state`
+  - `:im.review/rating`
+  - `:im.review/createdAt`
+  - `:im.review/author`
+  - `:im.review/subject`
+  - `:im.review/content`
+- Optional attributes - only include the key if it has a non-empty value
+  - `:im.review/listing`
+
+Supported references
+
+- `#im.review/listing` -> `#im.listing`
+- `#im.review/author` -> `#im.user`
+- `#im.review/subject` -> `#im.user`
+
+Example syntax
+
+```
+[[:im.review/id]
+  #:im.review{:content "Exactly as advertised. Bummed this was a one time deal."
+              :rating 5
+              :type :review.type/ofProvider
+              :state :review.state/public
+              :createdAt #inst "2018-01-06T00:10:10Z"
+              :author #im/ref :user/jane
+              :subject #im/ref :user/john
+              :listing #im/ref :listing/rock-sauna}]
+```
+
+### Value types
+
+Intermediary format uses some notable value types:
+
+- [Location](#location)
+- [Money](#money)
+- [UUID](#uuid)
+- [Timestamp](#timestamp)
+
+#### Location
+
+Location is given as
+[LatLng type](https://docs.mapbox.com/android/maps/api/9.6.0/com/mapbox/mapboxsdk/geometry/LatLng.html)
+representing the latitude and longitude of the location. Both latitude
+and longitude are 32 bit floats. The `location` attribute is used in the
+FTW template to provide listing location coordinates through either
+Mapbox or Google maps.
+
+If your source data has addresses specified, and you would like to use
+them to determine locations, you can use e.g. the Mapbox
+[forwardGeoCode](https://github.com/sharetribe/ftw-daily/blob/01cdf40368380d64d90136efb37948c1da79ee0e/src/components/LocationAutocompleteInput/GeocoderMapbox.js#L106)
```

Comment:
Our code may be subject to change so this link may expire. However, it's good to mention that if the source data only has addresses, then how to turn those into coordinates. Maybe we could mention that coordinate information can be fetched for addresses using a 3rd party geocoding API and link to few options, for example Google and Mapbox, here.